src/views/call-details/components/StreamCard.tsx

```diff
@@ -110,8 +110,8 @@ const StreamCard: React.FC<StreamCardProps> = (props) => {
 
   const classes = ['streamCard', getConnectionClass(stream), expanded ? 'expanded' : ''];
   const avatarSize = 112;
-  const avatarIcon = avatartImage ? (
-    <img src={avatartImage} style={{ width: avatarSize, height: avatarSize }} onError={() => setAvatartImage('')} />
+  const avatarIcon = stream.photo ? (
+    <img src={stream.photo} style={{ width: avatarSize, height: avatarSize }} onError={() => stream.photo} />
```

Comment:
**nitpick(blocking)**: If the image component has an error while trying to load the image, the error callback will set the stream photo instead of setting an undefined photo. Have you thought/analyzed what we should do in this scenario?